mmocr/core/__init__.py

```diff
@@ -1,3 +1,4 @@
+from .det_recog_visualize import *  # noqa: F401, F403
```

Comment:
Since you `import *` from  det_recog_visualize, the internal functions should be prefix-ed with `_`.